zepid/sensitivity_analysis/Simple.py

```diff
@@ -5,41 +5,44 @@
 
 
 class MonteCarloRR:
+    """Monte Carlo simulation to assess the impact of an unmeasured binary confounder on the results
```

Comment:
`r"""`?

Reply:
we only need `r` if there is some python escape sequence, like `\f` (which shows up in latex)